src/ixautomation/vms/.templates/kvm_truenas_install.xml

```diff
@@ -199,7 +199,7 @@
     <input type='keyboard' bus='ps2'>
       <alias name='input2'/>
     </input>
-    <graphics type='spice' port='5900' autoport='yes' listen='127.0.0.1'>
+    <graphics type='vnc' port='5900' autoport='yes' listen='127.0.0.1'>
```

Comment:
Should the port be "-1" and have keymap='en-us' be present here like the others?

Reply:
Yeah miss that one.